src/pricingutil.cpp

```diff
@@ -0,0 +1,13 @@
+#include "pricingutil.h"
+
+float PricingUtil::calcVal(float prevPrice, float interest, float oleoConstant) {
+    float result = (prevPrice * (0.9 + interest)) * oleoConstant;
+    this->val = result;
+    return result;
+}
+
+float PricingUtil::getVal() {
+    return this->val;
+}
+
+PricingUtil::PricingUtil() {}
```

Comment:
initialize val here please

Reply:
Hey Chris I just fixed this under a new PR called step-1-gavin